tools/cli/internal/cli/merge/merge.go

```diff
@@ -65,6 +67,15 @@ func (o *Opts) PreRunE(_ []string) error {
 	return err
 }
 
+func prettyPrint(data []byte) error {
```

Comment:
NIT: Not sure about the name of this function. What about `prettyPrintJson` and `jsonBytes []byte` as argument?

Reply:
updated